kpi/deployment_backends/kobocat_backend.py

```diff
@@ -1009,27 +1008,27 @@ def __validate_bulk_update_payload(payload: dict) -> dict:
         """
         if not 'submission_ids' in payload:
             raise KobocatBulkUpdateSubmissionsClientException(
-                detail=_('`submission_ids` must be included in the payload')
+                detail='`submission_ids` must be included in the payload'
             )
 
         if not isinstance(payload['submission_ids'], list):
             raise KobocatBulkUpdateSubmissionsClientException(
-                detail=_('`submission_ids` must be an array')
+                detail='`submission_ids` must be an array'
             )
 
         if len(payload['submission_ids']) == 0:
             raise KobocatBulkUpdateSubmissionsClientException(
-                detail=_('`submission_ids` must contain at least one value')
+                detail='`submission_ids` must contain at least one value'
             )
 
         if not 'data' in payload:
             raise KobocatBulkUpdateSubmissionsClientException(
-                detail=_('`data` must be included in the payload')
+                detail='`data` must be included in the payload'
             )
 
         if len(payload['data']) == 0:
             raise KobocatBulkUpdateSubmissionsClientException(
-                detail=_('Payload must contain data to update the submissions')
+                detail='Payload must contain data to update the submissions'
```

Comment:
I think, this comment is returned to users. We need to keep the translation.
All this code has been removed anyway in  basic case management branch. Would you mind to keep it to avoid conflicts on merge?